### ISSUE TITLE
NOT RESPONDING, HUGE SCENE EXPORTS

### ISSUE THREAD

**Issue by FireCulex** [obsproject/obs-studio#5058]
### Operating System Info

Other

### Other OS

Windows 11

### OBS Studio Version

27.0.1

### OBS Studio Log URL

https://obsproject.com/tools/analyzer?log_url=https%3A%2F%2Fobsproject.com%2Flogs%2FJSC9sdvNWUQsN3lx

### Expected Behavior

Not to write a 500mb scene, not to constantly lag and stop responding when opening obs, closing obs, switching scene's, changing anything, export less than ~100k json

### Current Behavior

Export scene results in 

yMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwntsYMIzp03CXNZDwkHfQMKQYD7Cz6s6wrYmOcJujTjCnEE4wlMjOMJOFTjCcAk4wmwEOMKRAjjCegI4wnoCOMJ6AjjCegI4wnoCOMJ6AjjCegI4wnoCOMJ6AjjCegI4wnoCOMJ6AjjCegI4wnoCOMJ6AjjCegI4wnoCOMJ6AjjCegI4wnoCOMJ6AjjCegI4wnoCOMJ6AjjCegI4wnoCOMKLzDnCpeg9whVJRMJD9EfCmY9MwrbpWcKw8mfCJn10wtIFgcKyq4bCxoGLwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwgAAyMIAAMjCAADIwg

between 250 and 500 megs depending on which scene was exported. 

### Steps to Reproduce

1. Create a New Scene
2. add a few sources
3. export the scene
4. 250-500mb json 

### Anything else we should know?

Game Mode is enabled.

**Comment by notr1ch** [comment]
Windows 11 is not an officially supported platform.

**Comment by FireCulex** [comment]
> 
> 
> Windows 11 is not an officially supported platform.

Had this problem last week in windows 10. Uses 3 gigs of ram and hd spikes. Shouldn't have mentioned it.

**Comment by WizardCM** [comment]
As a side note, I just tested on Windows 10 and built a 9KB scene collection export.

**Comment by FireCulex** [comment]
> 
> 
> As a side note, I just tested on Windows 10 and built a 9KB scene collection export.

Yup, I did too, I'm not sure what the determining factor is, trying to build the scene for the third time one by one and export by each source.

**Comment by FireCulex** [comment]
![explorer_CKr9EL38Kv](https://user-images.githubusercontent.com/9983630/128001083-a7772d32-30d1-4839-bda8-a17225d63346.png)

Looks like the actual scene files can get corrupted.

**Comment by FireCulex** [comment]
Upon inspection of one of the jsons there's a lot of chunk_data I believe is related to VST Filters, perhaps that is the problem.

"plugin_path":"C:/Program Files/VSTPlugins/ReaPlugs/reacomp-standalone.dll"},"sync":0,"versioned_id":"vst_filter","volume":1.0},{"balance":0.5,"deinterlace_field_order":0,"deinterlace_mode":0,"enabled":false,"flags":0,"hotkeys":{},"id":"noise_gate_filter","mixers":255,"monitoring_type":0,"muted":false,"name":"Noise Gate","prev_ver":452984833,"private_settings":{},"push-to-mute":false,"push-to-mute-delay":0,"push-to-talk":false,"push-to-talk-delay":0,"settings":{},"sync":0,"versioned_id":"noise_gate_filter","volume":1.0},{"balance":0.5,"deinterlace_field_order":0,"deinterlace_mode":0,"enabled":false,"flags":0,"hotkeys":{},"id":"compressor_filter","mixers":255,"monitoring_type":0,"muted":false,"name":"Compressor 2","prev_ver":452984833,"private_settings":{},"push-to-mute":false,"push-to-mute-delay":0,"push-to-talk":false,"push-to-talk-delay":0,"settings":{"attack_time":1,"output_gain":3.0,"ratio":2.0,"release_time":40,"threshold":-8.0},"sync":0,"versioned_id":"compressor_filter","volume":1.0},{"balance":0.5,"deinterlace_field_order":0,"deinterlace_mode":0,"enabled":false,"flags":0,"hotkeys":{},"id":"compressor_filter","mixers":255,"monitoring_type":0,"muted":false,"name":"Compressor 3","prev_ver":452984833,"private_settings":{},"push-to-mute":false,"push-to-mute-delay":0,"push-to-talk":false,"push-to-talk-delay":0,"settings":{"output_gain":3.0},"sync":0,"versioned_id":"compressor_filter","volume":1.0},{"balance":0.5,"deinterlace_field_order":0,"deinterlace_mode":0,"enabled":false,"flags":0,"hotkeys":{},"id":"gain_filter","mixers":255,"monitoring_type":0,"muted":false,"name":"Gain","prev_ver":452984833,"private_settings":{},"push-to-mute":false,"push-to-mute-delay":0,"push-to-talk":false,"push-to-talk-delay":0,"settings":{"db":9.0},"sync":0,"versioned_id":"gain_filter","volume":1.0},{"balance":0.5,"deinterlace_field_order":0,"deinterlace_mode":0,"enabled":false,"flags":0,"hotkeys":{},"id":"limiter_filter","mixers":255,"monitoring_type":0,"muted":false,"name":"Limiter","prev_ver":452984833,"private_settings":{},"push-to-mute":false,"push-to-mute-delay":0,"push-to-talk":false,"push-to-talk-delay":0,"settings":{"release_time":100,"threshold":-3.0},"sync":0,"versioned_id":"limiter_filter","volume":1.0},{"balance":0.5,"deinterlace_field_order":0,"deinterlace_mode":0,"enabled":false,"flags":0,"hotkeys":{},"id":"vst_filter","mixers":255,"monitoring_type":0,"muted":false,"name":"VST 2.x Plug-in","prev_ver":452984833,"private_settings":{},"push-to-mute":false,"push-to-mute-delay":0,"push-to-talk":false,"push-to-talk-delay":0,"settings":{"chunk_data":"dGZmcAAAAQCzTwAACnajMKcEtj/L6JHPC4IawL50LggqHP0/qcK33Dw6NcAvID2Uyf03QDw0GypsZ5pAAAAAAJokIkGk5Jz182aaQAAAAAAyeidBYOJU/gAAAACHdwAAAAAAAAAAAAAAcOdAAQAAAegDAAAAYIxEACCCRADAg0QAgB1EAMBtRAAASEQAIMlEAKCLRABAi0QAYKxEAMAeRABg2EQAILdEAICrRAAAsEQAwJ9EAIBBRACQAkUAAGhEAGDVRABgmUQAQNxEAGC3RABAB0UAgBRFAOC7RACgz0QAAPhEALAeRQAgCkUAwMBEABBKRQBAu0QAIKtEALAKRQAA1UQAQPxEAMDxRADgDUUAUABFAEBNRQBgoUQAANtEAJArRQCwJEUAABtFAADYRADwM0UAECpFAKB0RQCgQEUAYCVFAFAYRQBQPEUAMGlFAEAvRQDwB0UAoDZFAFCSRQBgoEUAkF9FAGBBRQAoi0UAYJJFAKCCRQBAUkUAQIdFAPBxRQCwhkUAsJJFALCfRQB4gUUAYLFFAFDARQBYvUUA0MtFAGjMRQAApEUAiKRFAMCARQDIsEUAOK1FALiqRQBAz0UA4J9FAACJRQAwskUA0LhFABiVRQBYskUAEFlFABi3RQAQlEUACL1FAHCpRQA4wUUAIJ1FABiRRQDIykUAUKNFAGipRQAItkUAuMdFAHipRQB4wEUA6NVFAHB2RQDAqEUAYJ1FANCoRQCw/UUAYKJFAAjBRQDw1kUAiM1FABCQRQBoqEUA4LtFAGCERQAw4EUAoMxFAOC9RQAIv0UAeNBFAHDPRQBQyEUA+NRFAPi/RQBw6UUACMRFABiRRQAYw0UAQLVFAJCpRQBowkUAUOJFAAimRQBgu0UAAMVFALCkRQDQ+0UAONFFAKDBRQAQ6UUA0OlFAHQARgDw7EUASNVFADjzRQBgq0UA+LhFALDiRQA4+EUA9AJGAIjTRQBo00UAUM5FANjNRQDYzUUAcMlFALDgRQCwzUUAIOlFAFDMRQB440UAWPZFAKAURgC8AEYA+ApGAHACRgCIBUYA1BVGADQfRgDoIkYATBxGAAQIRgDoDUYAaAxGAAAJRgA0EUYApA1GAOgjRgBwBUYAgCZGANQuRgBIJEYASB9GABAzRgC4NEYA5DZGACQrRgBcJUYACDBGAKwsRgDQRUYAcEdGAJwvRgBwWkYAyDNGAJBRRgA8QUYAaFlGALRDRgBEZUYAwE5GADxeRgBsQUYAqERGAFw5RgAYT0YAoE1GADxgRgAMQEYApE9GADBDRgC0OUYA7E5GAJA6RgCcUUYA3DhGAKRWRgBgSUYA0FZGADhdRgD4TEYAnFpGAMhtRgAkZkYAOH5GAAReRgCkZ0YAeFpGALRdRgBQVEYAjFRGAAxFRgAQSUYABFhGADhPRgBQWkYAcFdGAIxvRgDofUYA4G1GAEhuRgDIaUYAZHhGACKARgAgaUYAgHFGAERoRgD+jUYA4HJGAGCBRgCQekYA4IpGAEqCRgD4fEYAGIRGAFCBRgBQgUYAAF1GAIR4RgCQaEYA1opGADaORgD+k0YAXJVGAGaxRgB+t0YAKsZGAAzPRgCW40YAnANHAI0DRwD8/EYA9gVHANIJRwA+EkcAuhpHABMZRwD7GUcAfxFHACsTRwD3CUcAdgNHAO8BRwB670YASNxGAIDqRgCc0kYAItBGAPTDRgDEuUYAerlGAJyvRgAgrkYAeKJGAE6cRgAwnkYAMqJGAICaRgBeikYAgqtGAKiORgAEk0YAPo5GAAyHRgDKhUYAOIJGAFyARgBQjUYA6IxGAB6XRgD+gEYARpxGAOiGRgDihkYAmoJGAFKHRgBqiEYArJNGAP6aRgBoi0YAxpVGALaJRgBklUYAIJhGAACbRgC0lEYASphGAJKeRgB0pUYAyKNGAKKiRgBcnkYAQLJGAJ6nRgAwrUYANLJGAKa1RgCMuUYAWrBGAGrFRgAGxkYARNFGADLNRgCaz0YApORGAATURgD62EYAnOFGAGLhRgAO5EYAAuNGABbmRgCS3EYAJt5GAObbRgBO10YAgOhGADTfRgDc90YAaOtGAFrtRgAe5kYAAPZGALDoRgDa20YABOJGAIjxRgCs6EYABuJGAKjjRgBY6EYAjQJHAFblRgAm5EYAOOxGAJbgRgBe60YA7MpGAOTVRgC2zEYAeNlGAJrSRgAiykYA+NVGAJ6+RgCKwEYAespGALq4RgBAxEYAwMFGALKuRgDQuUYA6KlGANyjRgBKs0YATKpGALC3RgDEuUYAVr1GAKS2RgAusUYA8KZGAOycRgB2rEYA2J9GAGKXRgDmj0YA9qNGAAqbRgD4k0YA1pZGALaiRgASlkYAPI9GABqfRgBIfUYAZJlGAJB/RgAAj0YAuIJGAFSNRgBwi0YAMpRGAB6ERgAamUYAQotGALSZRgBslEYATJZGADiVRgCShkYALptGAKKKRgBoj0YA8ppGABaaRgA2lEYA9oxGAL6VRgCijEYA0JlGAHCMRgDYj0YAeppGAJ6gRgDSmUYAQKBGAHagRgDqlUYAEI1GAJKNRgA2oEYAAHlGAPiMRgBGhkYA2qNGAO6cRgDkk0YAMJhGAHadRgBOjUYAYH5GALB8RgAkg0YA1I9GAGqGRgBUjUYAlJVGADh+RgCQlUYA8HdGAGKDRgDAhEYARH5GAKCDRgAOgkYAhH5GACxuRgAgY0YAVHxGAKR8RgCkPUYAGoJGAD6CRgDMaUYA2G1GANRnRgCYcEYAwFtGAMhLRgCcckYArD5GAMxWRgAMWUYA2ElGAIw6RgAEOkYAbEhGADA/RgC0O0YAOD1GAOw7RgDsNEYAnDNGABAyRgDcKUYAhCNGAOwpRgBgEEYArBhGANQXRgAECUYAJCxGAJwGRgCUAEYACPVFAFD6RQAQAUYAiN1FABjrRQBo9UUAcMdFAADfRQBAz0UAOLVFAMCpRQAQ1EUAQMtFAKC2RQCorEUAINRFAICsRQA4lkUAQLVFANB2RQCgnkUAWKRFAFiIRQCAg0UAYHhFAGiARQAwb0UAAHFFAKiMRQCQQ0UAAE5FALAgRQBgTEUAEClFAHBJRQCAIEUAQA1FAFAHRQDgBUUAIPVEAJAaRQBA4EQA4PBEAIDKRADAs0QAYKREACCmRADApkQAYKFEAKCoRACAuEQAQHNEACCARAAAXkQAwGFEAEBO

**Comment by WizardCM** [comment]
Yep, sounds like a VST not storing data efficiently. Unfortunately we have no control over

**Comment by FireCulex** [comment]
Throw it out? Give chunk_data a size limit so plugins that misbehave don't get their values stored altering the overall OBS experience. 
I actually fixed it, I disabled microphone in settings and it went from 435mb to 18kb.  hoorah! OBS is so responsive now. On the other profile I just removed the reacomp vst plugin from mic settings and it shrunk from ~250mb to 37kb. Awesome!
I can re-add reacomp and its still small so I don't know what has to occur for the chunk_data to get out of control, but I do know its OBS that's saving it even if it's actually a bug in the plugin.

**Comment by Fenrirthviti** [comment]
Going to reopen this, we should probably have some kind of sanity check when saving plugin/VST data in general with at least a warning to the effect of like "Profile data is exceeding 25mb, this can indicate a plugin might be saving more data than necessary" or something similar.

**Comment by Warchamp7** [comment]
Closing as this is a very old issue and appears to be a problem specific to the YouLean Loudness VST itself
